scripts/popup.js

```diff
@@ -90,6 +90,12 @@
       }
     }).appendTo(this.$popup);
 
+    if (!header) {
+      self.$popupContent.css({
```

Comment:
@larsmagu Would it be possible to change the padding by adding an extra class? Customizing the look of H5P content via stylesheet overrides becomes more difficult with every piece of styling that's set in JavaScript instead of CSS.

Reply:
@larsmagu Cool, thanks a lot!